frontend/apps/client/src/features/my-calendar/ui/CalendarCardList/index.tsx

```
@@ -1,78 +1,84 @@
-                start={start}
-              />
-              <DefaultCard
-                card={card}
-                end={end}
-                start={new Date(ey, em, ed, 0, 0)}
-              />
-            </div>
-          );
-        }
+const groupByOverlap = (dayCards: CardItem[]) => dayCards.reduce((acc, cur, i, arr) => {
+  if (i === 0 || arr[i - 1].end <= cur.start) return [...acc, [{ ...cur, idx: 0 }]];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)